src/components/stats/SoloCommits.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+
+import { Query } from 'react-apollo';
+import { gql } from 'apollo-boost';
+import client from '../../utils/GitHubGQL';
+
+class SoloCommits extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      startDate: new Date(props.startDate).toISOString(),
+      endDate: `${props.endDate}T23:59:59.999Z`,
+      username: props.username,
+      repository: props.repository,
+      winner: '',
+      highScore: 0,
+    };
+  }
+
+  componentDidMount() {
+    const { username } = this.state;
+    const { repository } = this.state;
+    const { startDate } = this.state;
+    const { endDate } = this.state;
+
+    client.query({
+      query: gql`
+      {
+        repository(owner: "${username}", name: "${repository}") {
+          defaultBranchRef {
+            target {
+              ... on Commit {
+                history(since: "${startDate}" until: "${endDate}") {
+                  nodes {
+                    additions
+                    author {
+                      user {
+                        login
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }`,
+    })
+      .then((result) => {
+        this.findHigest(result);
+      });
+  }
+
+  findHigest(data) {
+    const testing = {};
+    const formatted = data.data.repository.defaultBranchRef.target.history.nodes;
+
+    formatted.forEach((el) => {
+      if (el.author.user != null) {
+        if (!testing[el.author.user.login]) {
+          testing[el.author.user.login] = el.additions;
+        } else {
+          testing[el.author.user.login] += el.additions;
+        }
+      }
+    });
+    // eslint-disable-next-line max-len
+    const highScoreWinner = Object.keys(testing).reduce((a, b) => (testing[a] > testing[b] ? a : b));
+    this.setState({ winner: highScoreWinner });
+    this.setState({ highScore: testing[highScoreWinner] });
+  }
+
+  render() {
+    const { winner } = this.state;
+    const { highScore } = this.state;
+
+    return (
+      <Query
+        query={gql`
+          {
+            user(login: "${winner}") {
```

Comment:
String interpolation not needed (remove brackets dollar and quotes

Reply:
GQL needs this